lib/http/publisher.ex

```diff
@@ -16,20 +16,22 @@ defmodule TelemetryInfluxDB.HTTP.Publisher do
     config
     |> Map.update!(:port, &:erlang.integer_to_binary/1)
     |> Map.put(:pool_name, pool_name)
+    |> Map.put(:backend_options, Map.take(config, [:ssl]) |> Enum.into([]))
```

Comment:
```suggestion
    |> Map.put(:backend_options, Map.get(config, :ssl) |> Enum.into([]))
```

Reply:
Здесь несоответствие -- `:ssl` присваивается всему `:backend_options`. Я бы сделал:

```suggestion
    |> Map.put(:backend_options, [
      ssl: Map.get(config, :ssl, [])
    ])
```